flocker/control/_persistence.py

```diff
@@ -37,6 +38,8 @@ def default(self, obj):
         elif isinstance(obj, FilePath):
             return {_CLASS_MARKER: u"FilePath",
                     u"path": obj.path.decode("utf-8")}
+        # elif isinstance(obj, UUID):
+        #    return unicode(obj)
```

Comment:
I was planning on relying on the `field` factory, but could do so if necessary, yeah.